docs/source/ruletypes.rst

```diff
@@ -571,6 +571,13 @@ ElastAlert will use ``fields`` to retrieve stored fields. Both of these are repr
 See https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-fields.html for more details. The fields used come from ``include``,
 see above for more details. (Optional, boolean, default True)
 
+scan_entire_timeframe
+^^^^^^^^^^^^^^^^^^^^^
+
+``scan_entire_timeframe``: If true, when ElastAlert starts, it will always start querying at the current time minus the timeframe.
+``timeframe`` must exist in the rule. This may be useful, for example, if you are using a flatline rule type with a large timeframe,
+and you want to be sure that if ElastAlert restarts, you can still get alerts. This may cause duplicate alerts for some rule types.
```

Comment:
can you add more color on what circumstances would cause this? and/or elaborate on _which_ rule types?